### PR TITLE
2nd Brno Docker Meetup Blog Post

### PR DESCRIPTION
**Pull request by bexelbie**

### CODE REVIEW

File: source/blog/2015-10-21-docker-brno-meetup-2.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+[Yury Tsarev](https://cz.linkedin.com/in/yurytsarev) from GoodData presented, ["Test Driven Infrastrucure with Docker, Test Kitchen and Serverspec"](https://drive.google.com/a/redhat.com/file/d/0B5OHcgvKZLcdcFJkbGZVQkZvTnM/view)  Yury strongly believes that infrastructure code should be treated like any other code.  This means apply a test driven development model, storing it in a source control system and building a regression test suite.  He suggests doing this with [Test Kitchen](kitchen.ci), a pluggable and extensible test orchestrator that originated in the Chef community.  Using Test Kitchen's [docker provider](github.  com/portertech/kitchen-docker), a docker container can be used to simulate a machine under test.  Then [Serverspec](serverspec.org) can verify that the configuration code, Puppet in Yury's case, properly setup the machine.  Shell mocking is used to bypass external dependencies and docker limitations.
```
Comment (ytsarev) [inline source/blog/2015-10-21-docker-brno-meetup-2.md]: Hi, there are some broken embedded links in this chunk
```
s#kitchen.ci#http://kitchen.ci#
s#github.  com/portertech/kitchen-docker#http://github.com/portertech/kitchen-docker#
s#serverspec.org#http://serverspec.org#
```
Thanks for a very nice writing

**Comment by bexelbie** [comment]
updated with changes from @ytsarev TY!

File: source/blog/2015-10-23-docker-brno-meetup-2.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+[Yury Tsarev](https://cz.linkedin.com/in/yurytsarev) from GoodData presented, ["Test Driven Infrastrucure with Docker, Test Kitchen and Serverspec"](https://drive.google.com/a/redhat.com/file/d/0B5OHcgvKZLcdcFJkbGZVQkZvTnM/view)  Yury strongly believes that infrastructure code should be treated like any other code.  This means apply a test driven development model, storing it in a source control system and building a regression test suite.  He suggests doing this with [Test Kitchen](http://kitchen.ci), a pluggable and extensible test orchestrator that originated in the Chef community.  Using Test Kitchen's [docker provider](github.com/portertech/kitchen-docker), a docker container can be used to simulate a machine under test.  Then [Serverspec](http://serverspec.org) can verify that the configuration code, Puppet in Yury's case, properly setup the machine.  Shell mocking is used to bypass external dependencies and docker limitations.
```
Comment (ytsarev) [inline source/blog/2015-10-23-docker-brno-meetup-2.md]: Thanks for the fixes, 
```
[docker provider](github.com/portertech/kitchen-docker)
```
still misses `http://`

**Comment by bexelbie** [comment]
additional url error noticed by Eagle-Eyes @ytsarev fixed.

**Pull request merged**